examples/zoomable-chart/yarn.lock

```diff
@@ -1 +0,0 @@
-../yarn.lock
```

Comment:
this was removed and readded automatically by running the script that removes references to unpm